index.html

```diff
@@ -0,0 +1,450 @@
+              fill="white"
+            />
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M14.1915 4.09963C14.2251 5.8827 13.3293 6.84705 12.4849 7.32976C11.6634 7.79942 10.4436 7.99072 8.99672 7.11308C8.28719 7.71497 7.62853 8.37397 7.028 9.08288C7.92575 10.5912 7.63785 11.8535 7.15498 12.6816C6.68324 13.4906 5.73141 14.2562 4.05248 14.2215C3.7407 15.0876 3.4988 15.9866 3.3335 16.9117C4.51798 17.5812 5.01438 18.4732 5.17673 19.2846H7.21722C7.51737 12.5459 13.1299 7.17387 20.0095 7.17387C27.0815 7.17387 32.8144 12.8504 32.8144 19.8528C32.8144 26.8552 27.0815 32.5318 20.0095 32.5318C18.9558 32.5318 17.9318 32.4057 16.952 32.1681H11.9469C12.2118 32.2484 12.4516 32.3571 12.6633 32.4782C13.491 32.9513 14.2736 33.9123 14.2168 35.6153C15.0782 35.9257 15.9723 36.1689 16.8926 36.3384C17.7641 34.7662 19.0313 34.3722 20.0092 34.3722C20.9749 34.3722 22.1523 34.8264 22.964 36.3675C23.8775 36.2084 24.7657 35.9767 25.6224 35.6788C25.5628 33.8492 26.4713 32.8652 27.3271 32.376C28.1711 31.8935 29.4352 31.7048 30.9341 32.6668C31.6298 32.0847 32.2774 31.4481 32.8703 30.7636C31.868 29.193 32.1586 27.8778 32.6564 27.024C33.147 26.1827 34.1568 25.3883 35.9635 25.4921C36.2652 24.6552 36.5017 23.7876 36.6669 22.8954C34.9862 22.0314 34.5698 20.741 34.5698 19.7506C34.5698 18.7787 35.0436 17.5903 36.6623 16.7856C36.4958 15.8937 36.258 15.0265 35.955 14.19C34.0647 14.273 33.0532 13.362 32.5532 12.5046C32.0649 11.6671 31.8745 10.4116 32.8536 8.92296C32.2597 8.23927 31.611 7.60353 30.9143 7.02241C29.3328 8.007 28.0084 7.71945 27.1481 7.2276C26.3088 6.7478 25.5158 5.76638 25.5975 4.01827C24.7403 3.72171 23.8517 3.49146 22.9379 3.33374C22.0658 4.9338 20.7876 5.33334 19.8029 5.33334C18.8443 5.33334 17.6772 4.88595 16.8662 3.37222C15.9462 3.54318 15.0525 3.78779 14.1915 4.09963Z"
+              fill="white"
+            />
+          </svg>
+          <span class="logo-text">
+            <span class="logo-text1 block text-2xl font-bold tracking-wider"
+              >Pie World</span
+            >
+            <span class="logo-text2 leading-3">Conference 2022</span></span
+          >
+        </a>
+        <ul class="flex gap-4 text-metal-grey">
+          <li>
+            <a href="#" class="hover:underline">About</a>
+          </li>
+          <li>
+            <a href="#" class="hover:underline">Privacy Policy</a>
+          </li>
+          <li>
+            <a href="#" class="hover:underline">Licensing</a>
+          </li>
+          <li>
+            <a href="#" class="hover:underline">Contact</a>
+          </li>
+        </ul>
+      </div>
+      <hr class="my-4 w-full" />
+      <span class="text-center text-sm"
+        >© 2022
+        <a
+          href="https://github.com/shahadat3669"
+          target="_blank"
+          class="hover:underline"
+          >Shahadat Hossain</a
+        >. All Rights Reserved.
+      </span>
+    </footer>
```

Comment:
**[OPTIONAL]**

- Your footer is great on desktop however, it's not user-friendly on mobile as one can't really tell if its the footer section due to the styling and alignment of the fir part of the footer. To improve this, you can choose to hide the first part on mobile and then make it visible on desktop, that way, it could be pleasant to the users. 

:point_right:  *This is just a suggestion, it's fine if ignored or updated*

Reply:
**[REQUIRED CHANGES ON README FILE]**


-  Hey, great job with your readme,  however, the microverse readme template has been update the this [current version](https://github.com/microverseinc/readme-template). Do kindly update your readme to conform to these changes and also, customize it to suit your current project.

Reply:
### PRESENTATION VIDEO

- Please, let's add the missing video presentation which should feature you telling users about the website, showing different screen sizes, and showing some piece of code from your env which you are proud of as this is a core part of the project requirement to have your project approved.
- **NOTE:** The link to the video should be added in the readme the way you added the live demo link.

*Please, refer to the project requirement to implement this missing part of the requirement following strictly the instructions* :pray: